docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -eu
+
+until mysql -h "${MYSQL_HOST}" -u ${MYSQL_USER} -p ${MYSQL_PASSWORD} ${MYSQL_DATABASE} -e 'select 1'; do
```

Comment:
```suggestion
until mysql -h "${MYSQL_HOST}" -u ${MYSQL_USER} -p${MYSQL_PASSWORD} ${MYSQL_DATABASE} -e 'select 1'; do
```
Bez tego wyświetla się Usage mysql